google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreImpl.java

```diff
@@ -99,12 +122,18 @@ static class ReadWriteTransactionCallable<T> implements Callable<T> {
     private volatile TransactionOptions options;
     private volatile Transaction transaction;
 
+    private final com.google.cloud.datastore.telemetry.TraceUtil.SpanContext parentSpanContext;
```

Comment:
Can we import these as headers they appear several times in this file?

Reply:
Not using this anymore.